ResearchKit/ActiveTasks/ORKRecorder.m

```diff
@@ -180,7 +180,7 @@ - (NSString *)recorderType {
 }
 
 - (NSString *)logName {
-    return [NSString stringWithFormat:@"%@_%@", [self recorderType], _recorderUUID];
```

Comment:
Should we use the *property syntax* as per the documentation? Otherwise it looks good.